networkx/classes/multidigraph.py

```diff
@@ -84,21 +84,23 @@ class MultiDiGraph(MultiGraph,DiGraph):
     Add one edge,
 
     >>> G.add_edge(1, 2)
+    0
```

Comment:
If someone is just reading this section of the documentation, it might be a little mysterious that the `add_edge` method returns a number. Maybe it would be helpful to have a note here or nearby that `add_edge` returns the new edge key? Not sure.